packages/components/link/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@douro-ui/link",
+  "version": "0.0.0",
+  "description": "Douro UI - Metyis Design System Link Component",
+  "main": "src/index.ts",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Douro-ui/design-system"
+  },
+  "author": "Metyis Porto",
+  "bugs": "https://github.com/Douro-ui/design-system/issues",
+  "license": "MIT",
+  "scripts": {
+    "build": "vite build"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.11.4",
```

Comment:
@JuniorDiasOliveira should it be peer dependency?

Reply:
yes, Jessica will change it, thanks!